peekingduck/pipeline/nodes/model/yolo_license_plate.py

```diff
@@ -12,21 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-License Plate Detection model
-"""
+"""🔲 License Plate Detection model."""
 
 from typing import Any, Dict
 
+import numpy as np
+
 from peekingduck.pipeline.nodes.model.yolov4_license_plate import lp_detector_model
 from peekingduck.pipeline.nodes.node import AbstractNode
 
 
 class Node(AbstractNode):  # pylint: disable=too-few-public-methods
     """Initializes and uses YOLO model to infer bboxes from image frame.
 
-    The YOLO node is capable of detecting objects from a single class (License
-    Plate). It uses YOLOv4-tiny by default and can be changed to using YOLOv4.
+    This customized YOLO node is capable of detecting objects from a single
+    class (License Plate). It uses YOLOv4-tiny by default and can be changed to
+    use YOLOv4.
```

Comment:
It actually uses v4 by default because v4tiny doesn't work that well, and only should be used if FPS is critical over accuracy.

Reply:
Changed